src/Breadcrumbs/index.js

```diff
@@ -11,75 +11,25 @@
  */
 
 /** @jsx jsx */
-import { css, jsx } from '@emotion/react'
+import { jsx } from '@emotion/react'
 import PropTypes from 'prop-types'
 import { Link as GatsbyLink } from 'gatsby'
-import '@spectrum-css/breadcrumb'
+import { Breadcrumbs, Item } from '@adobe/react-spectrum'
 
-const Breadcrumbs = ({ selectedTopPage, selectedSubPages, ...props }) => (
-  <nav aria-label='Breadcrumb' role='navigation' {...props}>
-    <ul
-      className='spectrum-Breadcrumbs'
-      css={css`
-        display: block;
-      `}
-    >
-      <li className='spectrum-Breadcrumbs-item'>
-        <GatsbyLink
-          className='spectrum-Breadcrumbs-itemLink'
-          to={selectedTopPage?.path}
-        >
-          {selectedTopPage?.title}
-        </GatsbyLink>
-        <svg
-          className='spectrum-Icon spectrum-UIIcon-ChevronRightSmall spectrum-Breadcrumbs-itemSeparator'
-          focusable='false'
-          role='img'
-          aria-hidden='true'
-        >
-          <path
-            d='M7 5a.747.747 0 00-.22-.53L2.54.23a.75.75 0 10-1.06 1.06L5.19 5 1.48 8.71a.75.75 0 101.06 1.06l4.24-4.24A.747.747 0 007 5z'
-            className='spectrum-UIIcon--large'
-          />
-          <path
-            d='M5.5 4a.747.747 0 00-.22-.53C4.703 2.862 3.242 1.5 2.04.23A.75.75 0 10.98 1.29L3.69 4 .98 6.71a.75.75 0 101.06 1.06l3.24-3.24A.747.747 0 005.5 4z'
-            className='spectrum-UIIcon--medium'
-          />
-        </svg>
-      </li>
-      {selectedSubPages &&
-        selectedSubPages.map((page, index) => (
-          <li className='spectrum-Breadcrumbs-item' key={index}>
-            <GatsbyLink
-              className='spectrum-Breadcrumbs-itemLink'
-              to={page.path}
-            >
-              {page.title}
-            </GatsbyLink>
-            <svg
-              className='spectrum-Icon spectrum-UIIcon-ChevronRightSmall spectrum-Breadcrumbs-itemSeparator'
-              focusable='false'
-              role='img'
-              aria-hidden='true'
-            >
-              <path
-                d='M7 5a.747.747 0 00-.22-.53L2.54.23a.75.75 0 10-1.06 1.06L5.19 5 1.48 8.71a.75.75 0 101.06 1.06l4.24-4.24A.747.747 0 007 5z'
-                className='spectrum-UIIcon--large'
-              />
-              <path
-                d='M5.5 4a.747.747 0 00-.22-.53C4.703 2.862 3.242 1.5 2.04.23A.75.75 0 10.98 1.29L3.69 4 .98 6.71a.75.75 0 101.06 1.06l3.24-3.24A.747.747 0 005.5 4z'
-                className='spectrum-UIIcon--medium'
-              />
-            </svg>
-          </li>
-        ))}
-    </ul>
-  </nav>
-)
+const ParliamentBreadcrumbs = ({ breadcrumbsPages, ...props }) => {
```

Comment:
Now that we've moved to using v3 Breadcrumbs, I'm not sure if having this component here is providing any value. I think we should deprecate it:
```suggestion
/**
 * Parliament Breadcrumbs are deprecated. Please use the React Spectrum v3 Breadcrumbs instead.
 * See the docs for details: https://react-spectrum.adobe.com/react-spectrum/Breadcrumbs.html.
 *
 * @deprecated
 */
const ParliamentBreadcrumbs = ({ breadcrumbsPages, ...props }) => {
  console.warn('Parliament Breadcrumbs are deprecated. Please use the React Spectrum v3 Breadcrumbs instead. See the docs for details: https://react-spectrum.adobe.com/react-spectrum/Breadcrumbs.html');

```

Reply:
Great suggestion!